diff --git a/src/config/wireplumber.conf b/src/config/wireplumber.conf
index 4d231a5b..8717ca3f 100644
--- a/src/config/wireplumber.conf
+++ b/src/config/wireplumber.conf
@@ -68,8 +68,8 @@ wireplumber.profiles = {
     support.settings = required
     support.log-settings = required
     metadata.sm-objects = required
-    hardware.audio = required
-    hardware.bluetooth = required
+    #hardware.audio = required
+    #hardware.bluetooth = required
     hardware.video-capture = required
     policy.standard = required
     #policy.role-priority-system = optional
